model/src/main/java/com/ge/predix/acs/rest/Parent.java

```diff
@@ -0,0 +1,75 @@
+package com.ge.predix.acs.rest;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import com.ge.predix.acs.model.Attribute;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+/**
+ * Represents a source of inherited attributes.
+ */
+@ApiModel(description = "Represents a source of inherited attributes.")
+public class Parent {
+    @ApiModelProperty(
+            value = "The unique identifier of the parent, e.g. \"tom@ge.com\" or \"/site/sanramon\".",
+            required = true)
+    private String identifier;
+
+    @ApiModelProperty(
+            value = "Restrictions on when a subject inherits parent attributes based on the resource accessed,"
```

Comment:
Clarify behavior when 'scopes' set has more than one attribute. (is there a test for this ?)